plugins/win-capture/game-capture.c

```diff
@@ -1640,7 +1670,8 @@ static void game_capture_tick(void *data, float seconds)
 			gc->error_acquiring = true;
 
 		} else if (!gc->capturing) {
-			gc->retry_interval = ERROR_RETRY_INTERVAL;
+			gc->retry_interval = ERROR_RETRY_INTERVAL*
```

Comment:
Style: Typically, you need to surround math operators by a single space on each side. Don't leave a trailing space at the end of the line.